website/docs/sql_ddl.md

```diff
@@ -0,0 +1,408 @@
+---
+title: SQL DDL
+summary: "In this page, we introduce how to create tables with Hudi."
+toc: true
+last_modified_at: 
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+The following are SparkSQL DDL actions available:
+
+# Spark Create Table
+:::note
+Only SparkSQL needs an explicit Create Table command. No Create Table command is required in Spark when using Scala or 
```

Comment:
what does scala or python mean? Can we be specific around the Spark apis, for e.g  "DataSource APIs (Batch or Streaming) via Scala or Python"